frontend/js/Cart.js

```diff
@@ -36,14 +37,15 @@ function addItemToCart(item) {
                     };
                 });
             } else {
-                // L'article n'est PAS présent dans le panier, on l'y ajoute
+                /** L'article n'est PAS présent dans le panier, on l'y ajoute */
```

Comment:
```suggestion
                /** 
                 * L'article n'est PAS présent dans le panier, on l'y ajoute 
                */
```
ou tu laisse comme c'était `//`

Reply:
Là en fait vu que c'est sur une ligne il me semblait que c'était ok comme ça
?

/** This is a description of the foo function. */
function foo() {
}
> vu sur la doc : https://jsdoc.app/about-getting-started.html